Wrapping/Generators/Python/Tests/findEmptyClasses.py

```diff
@@ -67,6 +67,10 @@ def isEmpty(o):
            "vnl_file_matrix",
            "vnl_file_vector",
            "vnl_fortran_copy",
+           "CosineWindowFunction",
```

Comment:
Those functions are not currently wrapped, so I don't think it is necessary to exclude them (at least for now).

Reply:
https://github.com/InsightSoftwareConsortium/ITK/blob/651ee50286385b5913dc7c34a8cb67fa3fcab85c/Modules/Core/ImageFunction/wrapping/itkWindowedSincInterpolateImageFunction.wrap#L5-L8

Reply:
Those functions are indeed wrapped here: https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/Core/ImageFunction/wrapping/itkWindowedSincInterpolateImageFunction.wrap#L7